### ISSUE TITLE
gmt batch : Verbose (job counting)

### ISSUE THREAD

**Issue by delaviel-anger** [GenericMappingTools/gmt#4774]
Not a really an issue but : 
```
batch [INFORMATION]: Job 0 of 3 completed [ 33.3 %]
batch [INFORMATION]: Job 1 of 3 completed [ 66.7 %]
batch [INFORMATION]: Job 2 of 3 completed [100.0 %]
```
the counting is not correct

**Comment by PaulWessel** [comment]
Thanks, not a problem but looks stupid and goes agains Rule no 1: Try not to look stupid.  Begin fixed by #4775.

**Comment by delaviel-anger** [comment]
> Rule no 1: Try not to look stupid.
I haven't signed for that !
